rskj-core/src/jmh/java/co/rsk/jmh/web3/BenchmarkWeb3.java

```diff
@@ -240,6 +240,29 @@ public void traceGet(BasePlan plan) throws BenchmarkWeb3Exception {
         plan.getWeb3Connector().traceGet(transactionHash, Stream.of("0x0").collect(Collectors.toList()));
     }
 
+    @Benchmark
+    @Timeout(time = 30)
```

Comment:
do we need these method-level timeouts (here and bellow)? just curious if there's any method that does consume a lot of time..